dashboard/templates/base.html

```diff
@@ -56,7 +51,9 @@
                     (commit), {{ git_version.branch }} (branch)
                 </td>
                 {% endif %}
+                {% if last_updated %}
                 <td style="text-align: left">Data last updated on {{last_updated|date:"m/d/Y P T"}}</td>
```

Comment:
I'd add an `{% else %}` message in here, like "Data has not been loaded"

Reply:
Okay.

Reply:
How's this? https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1300/commits/b5df0747f4bc3da88d626cd1ee6877e9307cde79